appserver/admingui/war/src/main/webapp/WEB-INF/faces-config.xml

```diff
@@ -40,82 +41,7 @@
         </locale-config>
         <el-resolver>com.sun.webui.jsf.faces.UIComponentELResolver</el-resolver>
     </application>
-
-<!--
-    <component>
-        <component-type>com.sun.enterprise.tools.admingui.Tail</component-type>
-        <component-class>com.sun.enterprise.tools.admingui.jsf.component.Tail</component-class>
-    </component>
--->
-
-<!--
-    <render-kit>
-        <renderer>
-            <component-family>com.sun.enterprise.tools.admingui.Tail</component-family>
-            <renderer-type>com.sun.enterprise.tools.admingui.Tail</renderer-type>
-            <renderer-class>com.sun.jsftemplating.renderer.TemplateRenderer</renderer-class>
-        </renderer>
-    </render-kit>
--->
-
-    <render-kit>
```

Comment:
Why empty  `render-kit` section here?

Reply:
I'm not sure why it was there, probably because somebody commented out the contents but left the `render-kit` empty. I deleted the whole `render-kit`, as it was empty. Do you think it might cause issues? I saw that everything was working well in the Admin Console.

Reply:
I'm sorry. It's already commented )